packages/entities/entities-gateway-services/src/components/GatewayServiceForm.vue

```diff
@@ -652,16 +883,29 @@ const showTlsVerify = computed((): boolean => {
 })
 
 const validateName = (input: string): void => {
+  // reset the errors
+  resetFormFieldErrors('name')
   preValidateErrorMessage.value = validators.utf8Name(input)
 }
 
-/**
- * Is the form submit button enabled?
- * If the form.fields and formFieldsOriginal are equal, always return false
- */
-const canSubmit = computed((): boolean =>
-  (isEditing.value && (JSON.stringify(form.fields) !== JSON.stringify(formFieldsOriginal))) || ((checkedGroup.value === 'url' && !!form.fields.url) ||
-  (checkedGroup.value === 'protocol' && !!form.fields.host)))
+const canSubmit = computed((): boolean => {
+  // If in edit mode, can submit only if there are changes
+  if (isEditing.value) {
+    return JSON.stringify(form.fields) !== JSON.stringify(formFieldsOriginal)
```

Comment:
for future reference: you can use [objectsAreEqual()](https://github.com/Kong/konnect-ui-apps/blob/main/apps/gateway-manager/src/helpers/helpers.ts#L193) (literally the exact same logic you are using)

Reply:
updated